### DISCUSSION TITLE
Documentation Generator

### DISCUSSION THREAD

**Discussion by skillcraft-io** [binaryk/laravel-restify#495] [Q&A]
I was wondering what people use t generate documentation for their APIs using this package if any

**Comment by glennjacobs** [comment]
I was wondering the same.

I know of this package https://scribe.knuckles.wtf/, but it would require each request to have a custom method as it uses DocBlocks. I guess it's doable.

E.g. 

```php

/**
 * Update a User.
 * 
 * @group Account management
 */
public function store(RestifyRequest $request)
{
    parent::store($request);
}
```
